### DISCUSSION TITLE
Addition of admin controls

### DISCUSSION THREAD

**Discussion by maertendMSFT** [Azure/CloudShell#232] [Ideas]
We are evaluating adding tenant admin controls for Cloud Shell.

The following features are under consideration:
- Disabling Cloud Shell
- Pre-provisioning Cloud Shell storage accounts (admin can create and link a storage account first time users) 
- Custom Cloud Shell images
- Activity logging
- Custom timeouts

**Comment by chrisking81** [comment]
@maertendMSFT - is there any update on this? My FSI customer needs to block access to this, and there really isn't a very good story at the moment.
